apps/www/_blog/2024-08-26-supabase-lw12-hackathon.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Supabase Launch Week 12 Hackathon'
+description: Build an Open Source Project over 10 days. 5 prize categories.
+author: craig_cannon,tyler_shukert
+image: lw12/hackathon/thumbnail.png
+thumb: lw12/hackathon/thumbnail.png
+launchweek: 12
+categories:
+  - launch-week
+tags:
+  - launch-week
+  - hackathon
+date: '2024-08-26'
+toc_depth: 2
+---
+
+Last week we concluded [Launch Week 12 with so many new updates](https://supabase.com/launch-week), but those who have been following launch weeks might have noticed that we were not running the hackathon we usually run. Do not worry though, because hackathon isn’t going anywhere! Instead of running it along the launch week, we are running it a few weeks after the launch week providing you the time to play around with the new updates before the hackathon starts. 
+
+The hackathon starts on Friday, September 13th at 09:00 am PT and ends on Sunday, September 22nd at 11:59 pm PT. You could win an extremely limited edition Supabase swag and add your name to the Supabase Hackathon Hall of Fame.
+
+For some inspiration, check out all the [winners from previous hackathons](https://supabase.com/blog/tags/hackathon).
+
+This is the perfect excuse to "Build in a weekend, scale to billions.” Since you retain all the rights to your submissions, you can use the hackathon as a launch pad for your new Startup ideas, side projects, or indie hacks.
+
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase in some capacity.
```

Comment:
```suggestion
- You have 10 days to build a new **Open Source** project using Supabase in some capacity
```

Reply:
Some list items have full stops, others don't. Probably best without? @dshukertjr

Reply:
Great call. Let's go with without the period 👍